repositories/repo.go

```diff
@@ -289,11 +293,20 @@ func (s *Repo) MutatePublication(id string, u *models.Person, muts ...Mutation)
 	}
 
 	for _, mut := range muts {
-		mutator, ok := s.config.PublicationMutators[mut.Op]
+		mutator, ok := s.config.PublicationMutators[mut.Name]
 		if !ok {
-			return fmt.Errorf("unknown mutation '%s'", mut.Op)
+			msg := fmt.Sprintf("unknown mutation '%s'", mut.Name)
+			if mut.Line != 0 {
```

Comment:
Why is the difference between a mutation with a line set and one without?

Reply:
Line is set if the source of the mutations is line-delimited like a file or a textarea to give better error messages

Reply:
ah human input versus machine induced input (e.g. cronjob)

Reply:
yes